src/components/inputs/Selector.tsx

```diff
@@ -71,6 +71,20 @@ export default function Selector({
     }
   }
 
+  useEffect(() => {
+    function closePopper(e: KeyboardEvent) {
+      if (e.key === 'Escape') {
+        setAnchorEl(null)
+      }
+    }
+
+    window.addEventListener('keydown', closePopper)
+
+    return () => {
+      window.removeEventListener('keydown', closePopper)
+    }
+  }, [])
+
```

Comment:
Would it make more sense to make this a custom hook instead? I think this logic is repeated for new task modal as well so.

```suggestion
import { useEffect } from 'react';

const useCloseOnEscape = (closeFunction: () => void) => {
  useEffect(() => {
    const closePopper = (e: KeyboardEvent) => {
      if (e.key === 'Escape') {
        closeFunction();
      }
    };

    window.addEventListener('keydown', closePopper);

    return () => {
      window.removeEventListener('keydown', closePopper);
    };
  }, [closeFunction]);
};

export default useCloseOnEscape;


...

const Selector = (...) => {
     useCloseOnEscape(() => setAnchorEl(null));
     
     return <... />
}

```